indico/modules/designer/placeholders.py

```diff
@@ -361,6 +362,20 @@ class RegistrationPhonePlaceholder(RegistrationPDPlaceholder):
     field = 'phone'
 
 
+class RegistrationPicturePlaceholder(RegistrationPDPlaceholder):
+    name = 'picture'
+    description = _('Picture')
+    is_image = True
+
+    @classmethod
+    def render(cls, registration):
+        picture_data = [d for d in registration.data
+                        if d.field_data.field.personal_data_type == PersonalDataType.picture]
+        if picture_data and picture_data[0].filename:
+            buf = picture_data[0].open()
+            return Image.open(buf)
```

Comment:
- I think `.open()` whould be used with a `with` block.
- What happens when rendering the template for a registration that has no picture (ie where this method here returns `None`)?

Reply:
Yes of course. What I meant is when the render method returns None